app.css

```diff
@@ -5,3 +5,48 @@
 *::before, *::after {
   box-sizing: border-box;
 }
+
+.octo-cat {
+  width: 90%;
+  border: 1px solid grey;
+}
+
+.big-pics {
+  max-width: calc(99% / 3);
+  max-height: auto;
+  display: inline-block;
```

Comment:
Don't use `inline-block` to grid things. Use the `float`s that we used out today in class.

This is what's causing the weird drop down when you try to use `calc(100% / 3)`.